hw/ip/edn/doc/_index.md

```diff
@@ -199,18 +199,18 @@ See the [CSRNG IP]({{< relref "hw/ip/csrng/doc" >}}) waveform section for the CS
 ##### Peripheral Hardware Interface - Req/Ack
 The following waveform shows an example of how the peripheral hardware interface works.
 This example shows the case where the boot-time mode in the ENTROPY_SRC block is enabled.
+This example also shows where the hold time of the edn_bus is N = 4, where N is the number of cycles edn_bus is valid.
```

Comment:
i guess this should be updated for pop-on-req?

Reply:
Pesky words...the waveform is correct anyway. Will fix.